apps/src/templates/sectionAssessments/assessmentDataShapes.js

```diff
@@ -15,23 +15,23 @@ export const questionDataPropType = PropTypes.shape({
 
 // Shapes for single student multiple choice tables
 
-// Represents an individual students response to a single
+// Represents an individual student's response to a single
 // multiple choice question.
 export const studentResponsePropType = PropTypes.shape({
   isCorrect: PropTypes.bool,
-  answers: PropTypes.string,
+  responses: PropTypes.string,
```

Comment:
Might be good to have a comment here with what these strings could look like: `A` or `A, C`, etc.